### DISCUSSION TITLE
Setup issue(testing Strapi)

### DISCUSSION THREAD

**Discussion by VardanMelik** [strapi/strapi#6556] [Legacy Topics]
Windows 8(OS). My actions:
1. npm install -g strapi@beta
2. npx create-strapi-app my-project --quickstart

And I have this issue. Please help me to fix. Thx.

![Strapi](https://user-images.githubusercontent.com/49842390/83812874-530ba500-a6cd-11ea-9dc3-278274759559.png)

**Comment by derrickmehaffy** [comment]
Hello,

I see you are wanting to ask a question that is not really a bug report or a feature request, questions should be directed to the [discussions](https://github.com/strapi/strapi/discussions). Please see the following contributing guidelines for asking a question [here](https://github.com/strapi/strapi/blob/master/CONTRIBUTING.md#reporting-an-issue).

I have converted this to a discussion for you, we try to keep Github clean and only for bug reports and feature requests.

**Reply by VardanMelik** [reply]
Hi,
Thank you. If someone can help to fix this issue it will be useful.
